api4/schema.graphqls

```diff
@@ -85,24 +90,40 @@ type ChannelMember {
 type User {
 	id: String!
 	username: String!
+	# deliberately omitting password
 	email: String!
+	authData: String
 	firstName: String!
 	lastName: String!
 	nickname: String!
+	emailVerified: Boolean!
 	isBot: Boolean!
 	isGuest: Boolean!
 	isSystemAdmin: Boolean!
 	createAt: Float!
+	updateAt: Float!
 	deleteAt: Float!
 	authService: String!
 	customStatus: CustomStatus
 	status: Status
 	props: StringMap!
 	notifyProps: StringMap!
-	lastPictureUpdateAt: Float!
+	lastPictureUpdate: Float!
+	lastPasswordUpdate: Float!
+	failedAttempts: Float!
 	locale: String!
 	timezone: StringMap!
 	position: String!
+	mfaActive: Boolean!
+	mfaSecret: String!
```

Comment:
does this leak the user's mfa secret to anyone?

Reply:
Fair point, will remove it. :+1: